spec/factories.rb

```diff
@@ -70,6 +70,15 @@
     status { "submitted_for_2i" }
   end
 
+  factory :step_by_step_page_claimed_for_2i, parent: :step_by_step_page_submitted_for_2i do
+    status { "submitted_for_2i" }
+    review_requester_id { create(:user).uid }
```

Comment:
I gave you this code originally but I think this would be better (and is used elsewhere):

```suggestion
    review_requester_id { SecureRandom.uuid }
```

Reply:
Updated as suggested.